### ISSUE TITLE
Space should open dropdowns on win in non-main window

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#5772]
https://forums.zotero.org/discussion/129504/cannot-expand-dropdown-menu-from-keyboard

By default, Alt+ArrowDown opens `menulist` entries on windows - that's the keypress that must be used in settings/advanced search/etc. But in the main window, we add explicit handling of Space/Enter keypresses. Probably no reason not to apply that to all dropdowns.

Partly mentioned in https://github.com/zotero/zotero/issues/5202
